src/components/footer/footer.ts

```diff
@@ -16,10 +16,6 @@ export class FooterComponent extends LitElement {
      <footer class="container as-footer">
         <section>
           <h2>Analog Studio's Footer</h2>
-          <article class="col-xs-12">
-           <h3>Copyright</h3>
-            <p class="copyright-text">&copy; ${STARTING_YEAR} - ${currentYear} Analog Studios</p>
-          </article>
           <article class="col-xs-12">
```

Comment:
https://github.com/AnalogStudiosRI/www.analogstudios.net/pull/60#discussion_r778207024
Reference to switch the socials above the copyright paragraph tag.